src/configs/data-volume.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: data
+provisioner: kubernetes.io/no-provisioner
+reclaimPolicy: Retain
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: data-pv
+spec:
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteMany
+  storageClassName: data
+  nfs:
+    path: /mnt/disks/ssd1/data
+    server: master
```

Comment:
The NFS server is now the Node2 (data), we should change this accordingly.
We need to come up with better naming convention on those nodes I think.